src/share/cxx/HyperviscosityFunctorImpl.hpp

```diff
@@ -187,8 +189,9 @@ class HyperviscosityFunctorImpl
 
     if (m_data.nu_top > 0) {
 
+      //for top 3 levels and laplace, there is trivial nu_ratio only
       m_sphere_ops.vlaplace_sphere_wk_contra<NUM_BIHARMONIC_LEV>(
-            kv, m_data.nu_ratio,
+            kv, 1.0,
```

Comment:
Not knowing the Fortran, should this be `1.0` instead of one of the `nu_ratio` variables?

Reply:
i am not sure what's the question: for top 3 levels we won't use nu_ratios, but they are not always 1... i followed fortran.

Reply:
Thanks for the clarification; feel free to merge

Reply:
Michael is correct that NaN would work better, but i still want to keep 1.0 here...